### DISCUSSION TITLE
동영상 촬영 예제코드 중에 질문이 있습니다.

### DISCUSSION THREAD

**Discussion by earthboom** [naverz/zepeto-studio-kor#923] [5. Q&A]
1. 캐릭터가 사용하는 ZepetoCamera를 recorderCamera로 사용해도 괜찮나요? 아니면 새로 카메라를 생성하여 써야 하나요?

1.5. 새 카메라로 테스트를 해보았는데
'B5G6R5_UNormPack16' doesn't support ReadPixels usage on this platform. Async GPU readback failed.
UnityEngine.GUIUtility:ProcessEvent (int,intptr,bool&)
에러 메시지가 출력되고 있습니다. 
(예제의 RenderTexture에 대입한 color format은 R8G8B8A8_UNORM 입니다.)

2. videoPlayerObject 라는 GameObject가 있는데, 할당 후엔 딱히 조작이 없습니다. 이 GameObject는 무엇인가요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요 문의 주신 내용에 답변 드립니다.

1. 제페토 카메라도 사용이 가능하고, 일반적인 카메라도 사용이 가능합니다.
다만 제페토 카메라를 사용할 경우 셀피와 같은 뷰는 불가능하므로, 스크린샷 예제에서도 셀피용 카메라는 일반적인 카메라를 사용하고 있습니다.

추후 동영상 촬영 예제도 제공할 예정이오니 조금만 기다려 주시기를 부탁 드립니다.

2.  videoPlayerObject는  WorldVideoRecorder.AddVideoPlayerComponent 함수에 사용할 GameObject 타입의 변수입니다.


추가로 문의 주신 오류에 관해서는 코드나 스크린샷을 보면 보다 정확하게 알 수 있을 것 같습니다.

**Reply by koldshine** [reply]
안녕하세요, 동영상촬영 예제소스는 어디서 볼수 있나요?

**Reply by Zepeto-TechEvangelist** [reply]
제페토 스튜디오 월드 공식 가이드에 있습니다.
https://studio.zepeto.me/guides/video-save-share-post

**Comment by earthboom** [comment]
안녕하세요.
WorldVideoRecorder.AddVideoPlayerComponent 의 반환값이 계속 null로 오는데, 혹시 프로젝트에 따로 옵션을 조정하거나 하는 다른 작업이 필요한 건가요?
